src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -21,6 +15,7 @@ import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
 
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 import { logError } from '@edx/frontend-platform/logging';
+import algoliasearch from 'algoliasearch';
```

Comment:
This import is changing from `'algoliasearch/lite'` to `'algoliasearch'`. (I personally don't know the difference without doing some googling).